src/components/DivElement.js

```diff
@@ -1,6 +1,15 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
-
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super(content);
+    this.content = content;
+    this.tag = 'div';
+  }
+  render() {
```

Comment:
No need to define `render()`. By inheriting from `HTMLElement`, this function is "inherited" and you don't need to redefine it. Woo, less work! 💯